exporter/exporterhelper/internal/persistent_storage_test.go

```diff
@@ -16,17 +16,16 @@ package internal
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"reflect"
-	"sync"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/require"
 	"go.uber.org/zap"
 
 	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter/exporterhelper/internal/testutils"
```

Comment:
Let's call it `testutil` since singular form for packages is usually recommended in go

```suggestion
	"go.opentelemetry.io/collector/exporter/exporterhelper/internal/testutil"
```